config/prow/config.yaml

```diff
@@ -61,11 +61,11 @@ deck:
           highlight_regexes:
           - timed out
           - "ERROR:"
-          - (FAIL|Failure \[)\b
+          - (FAIL|Failure \[|FAILED)\b
           - panic\b
           - ^E\d{4} \d\d:\d\d:\d\d\.\d\d\d]
-          # This highlights the start of bazel tests/runs to skip go importing noise.
-          - "^INFO: Analyzed \\d+ targets"
+          # This highlights the start of tests to skip noise in build log
+          - "^> (Test|Test Cover|Integration Tests|E2E Tests)$"
```

Comment:
How about adding these regexes too:
- `level=fatal\b`
-  `fatal:`
- `make:.*Error 2`

The first one highlights errors like [here in line 195](https://prow.gardener.cloud/view/gs/gardener-prow/logs/ci-prow-autobump/1524139929267867648)
The second one highlights fatal errors like [here in line 22](https://prow.gardener.cloud/view/gs/gardener-prow/pr-logs/pull/gardener_gardener/5939/pull-gardener-integration/1524256418285228032) and hopefully those missing rebase errors too. I can't find an example in the list of failed tests right now
The third one covers generic make errors like [here in line 38](https://prow.gardener.cloud/view/gs/gardener-prow/pr-logs/pull/gardener_gardener/5938/pull-gardener-verify-image-build/1524241290558967808)

Reply:
Very good suggestions, added them to the list. Thanks! 🚀